test/correctness/simd_op_check.cpp

```diff
@@ -1889,6 +1889,9 @@ struct Test {
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1))*2 + i32(in_i16(x)) + i32(in_i16(x + 1)));
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1)) + i32(in_i16(x)) + i32(in_i16(x + 1)));
 
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 2*i32(in_u8(x - 1)) + 3*i32(in_u8(x)) + 5*i32(in_u8(x + 1)) + 7*i32(in_u8(x + 2)));
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 9*i32(in_u8(x - 1)) + 8*i32(in_u8(x)) + 7*i32(in_u8(x + 1)));
```

Comment:
Yes it is an immediate value {0,1}. The constant value is passed as an argument in the call at L1846, L1841 (https://github.com/halide/Halide/pull/2095/files#diff-007baef34b5299fdf08df3e32378726eR1846) with implementation for vrmpy in hvx_128.ll.